src-docs/src/views/guidelines/writing_guidelines.js

```diff
@@ -205,15 +205,27 @@ export const WritingGuidelines = () => {
           </GuideRuleExample>
         </GuideRule>
 
-        <GuideRule heading="Title case for products, solutions, and apps">
+        <GuideRule
+          heading="Title case for proper nouns"
+          description="Proper nouns include product names, solutions, apps, feature tiers, and subscription levels. "
+        >
           <GuideRuleExample
             type="do"
-            text="Product and solution names are always capitalized."
+            text="Product and solution names are always capitalized. Same goes for feature tiers and subscription levels."
             minHeight={200}
           >
-            <EuiTitle size="xs">
-              <span>Elastic APM</span>
-            </EuiTitle>
+            <EuiText size="xs">
+              <span>
+                <ul>
+                  <li>Welcome to Elastic Observability</li>
+                  <li>Elastic APM</li>
+                  <li>You have chosen Security Essentials.</li>
+                  <li>
+                    This feature is available for Gold subscriptions and higher.
+                  </li>
+                </ul>
+              </span>
```

Comment:
minor nit: you can lose the extra span wrapper if you're not using `EuiTitle`
```suggestion
              <ul>
                <li>Welcome to Elastic Observability</li>
                <li>Elastic APM</li>
                <li>You have chosen Security Essentials.</li>
                <li>
                  This feature is available for Gold subscriptions and higher.
                </li>
              </ul>
```

Reply:
Thanks!!